connect-examples/v2/node_subscription/bin/script/seed-data.js

```diff
@@ -105,9 +106,15 @@ async function addSubscriptionPlans() {
     batches: batches,
   };
 
+  // set the currency for the current merchant
+  const merchant = await merchantsApi.listMerchants();
+  const currency = merchant.result.merchant[0].currency;
+
   // Iterate through each item in the sample-seed-data.json file.
   for (const key in sampleData) {
     const currentCatalogItem = sampleData[key];
+    // Set the currency for each see data
+    currentCatalogItem.data.subscriptionPlanData.phases.forEach(x => x.recurringPriceMoney.currency = currency);
```

Comment:
[minor] I recommend a different variable name instead of `x`. Maybe `phase` in this case?